src/Util.cpp

```diff
@@ -572,6 +577,84 @@ int get_llvm_version() {
     return LLVM_VERSION;
 }
 
+#ifdef _WIN32
+
+namespace {
+
+struct DeferredFunction {
+    const std::function<void()> *run;
```

Comment:
I assume that there is no ownership implied and pointer is assumed to remain valid? (If so, make a comment to that effect might be good)

Reply:
It's really just a parameter pack. Maybe changing the name to GenericFiberArgumentsPack would communicate that better.

Reply:
But that's right, there's no ownership implied here. All that stuff is pointers to automatic storage

Reply:
I fixed up the names and changed it to a const-ref, so it's more clear there's no ownership.